docs/source/example_notebooks/dowhy_simple_example.ipynb

```diff
@@ -17,6 +17,10 @@
    "metadata": {},
    "outputs": [],
```

Comment:
good practice to commit a notebook without the output, only the code. Can you remove the output and then submit this notebook?

Reply:
actually we need to do this for all dowhy notebooks, but let's start with this one.